### DISCUSSION TITLE
How to record the process of simulation?

### DISCUSSION THREAD

**Discussion by parabolang** [sofa-framework/sofa#4284] [Write / Run a simulation] [answered]
Hi,
I wanted to record a video when I was running a simulation in SOFA. However, after I pressed 'v' in keyboard. I didn't see the video.
The error in the terminal report that,
[INFO]    [VideoRecorderFFMPEG] Start recording to /Desktop/DefrostSofaBundle_linux_python3.10_v22.06.01/screenshots/simple__r60_0001.mp4 ( yuv420p, 60 FPS, 5120000 b/s) using ffmpeg
sh: 1: ffmpeg: not found

I run script by sofapython3, and I have downloaded the ffmpeg in python environment.
Thanks for your help in advance!!!

**Comment by harshjn** [comment]
I have the same problem as recording a video didn't work for me either. Currently on linux, I'm taking screenshots periodically via xwininfo.

**Reply by parabolang** [reply]
Oic. I use the built-in method (Ctrl + Alt + Shift + R) in Linux instead.

**Reply by bakpaul** [reply, marked as answer]
Installing ffmpeg through python will not necessary install it for your whole OS. Actually this part of the GUI is not using the pythons library but the OS libraries, meaning that they must be reachable through your OS's environment variables. For linux they are `LD_LIBRARY_PATH` and `PATH`. If those paths doesn't contain the path to your pythons libraries, it is normal that it isn't working. 

To be able to use it, the easiest way on linux would be to use apt package management tool and install ffmpeg. This should work well.

**Reply by parabolang** [reply]
Thanks, Paul!!! 
I tried it just now. It works.
